### PR TITLE
better grammar in request.md document.

### PR DESCRIPTION
**Pull request by burdettmary**
this patch suggest better grammar changes in common write request structure documentation.

### CODE REVIEW

**Comment by ashcherbakov** [comment]
Can you please DCO signature?  https://github.com/hyperledger/indy-node/pull/1495/checks?check_run_id=284371652

**Review by brentzundel — APPROVED** [review]

**Review by esplinr — APPROVED** [review]

**Review by askolesov — APPROVED** [review]

File: docs/source/requests.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ Each Request (both write and read) is a JSON with a number of common metadata fi
  
 - `reqId` (integer): 
 
-    Unique ID number of the request with transaction.
+    Unique ID number of the requested transaction.
```
Comment (skhoroshavin) [inline docs/source/requests.md:135]: Same problem as in https://github.com/hyperledger/indy-node/pull/1497

**Review by skhoroshavin — CHANGES_REQUESTED** [review]

File: docs/source/requests.md (hunk 1 above)
Comment (brentzundel) [reply docs/source/requests.md:135]: This was the wording we settled on for #1497, does the same work here?
```suggestion
    Unique ID number of the request containing a transaction to be written.
```

**Comment by cjhowland** [comment]
@burdettmary Could you provide a brief update on this PR? With recent branch changes and the current development process on Indy Node/Plenum, we are wondering if these changes are still viable. If so, please close this PR and retarget it to `main`.

We will discuss and determine which PRs should be merged or closed at the next Indy Contributors Working Group meeting on May 9, 2023.

**Comment by WadeBarnes** [comment]
New documentation is being drafted on the `main` branch.

**Pull request closed without merging**